test/unit/visitor/inline.cpp

```diff
@@ -27,7 +27,7 @@ using nmodl::parser::NmodlDriver;
 // Procedure/Function inlining tests
 //=============================================================================
 
-std::string run_inline_visitor(const std::string& text) {
+std::string run_codegen_helper_visitor(const std::string& text) {
```

Comment:
For this file the name of the function could be kept as `run_inline_visitor`, since we don't use the `CodegenHelperVisitor`?

Reply:
I was mistaken, I thought this was in codegen_helper test file.